test/e2e/backup_test.go

```diff
@@ -49,14 +50,61 @@ var _ = Describe("[Restic] Velero tests on cluster using the plugin provider for
 
 	})
 
-	Context("When kibishii is the sample workload", func() {
-		It("should be successfully backed up and restored", func() {
+	When("kibishii is the sample workload", func() {
+		It("should be successfully backed up and restored to the default BackupStorageLocation", func() {
 			backupName = "backup-" + uuidgen.String()
 			restoreName = "restore-" + uuidgen.String()
 			// Even though we are using Velero's CloudProvider plugin for object storage, the kubernetes cluster is running on
 			// KinD. So use the kind installation for Kibishii.
-			Expect(RunKibishiiTests(client, cloudProvider, veleroCLI, veleroNamespace, backupName, restoreName)).To(Succeed(),
+			Expect(RunKibishiiTests(client, cloudProvider, veleroCLI, veleroNamespace, backupName, restoreName, "")).To(Succeed(),
 				"Failed to successfully backup and restore Kibishii namespace")
 		})
+
+		It("should successfully back up and restore to multiple BackupStorageLocations with unique credentials", func() {
```

Comment:
This test name could be more precise and helpful. By itself it is not operating with multiple BSLs, only an additional one besides the existing default. So maybe if it's named "should successfully back up and restore to an additional BackupStorageLocation with unique credentials" it'd avoid people looking for more than one BSL being used in this test (I have no idea who'd do that... maybe me :) ).

These are such nits:
s/additionalBsl/additionalBSL
s/additionalBslCredentials/additionalBSLCredentials
s/additionalBslBucket/additionalBSLBucket
etc

Thank you.

Reply:
Done :)